### ISSUE TITLE
aria live region updated to blank line when balloon pushes against wall

### ISSUE THREAD

**Issue by jbphet** [phetsims/balloons-and-static-electricity#238]
If the balloon is moved using accessibility features, when it pushes against the wall it is set to be a blank value.  It should probably say something like "balloon prevented from moving by wall".

To duplicate, go to http://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.1.6-dev.2/balloons-and-static-electricity_en.html?showLiveOutput (note the query parameter that enables output of live region updates to the console).  Make sure the console is visible.  Use the tab key to focus on the balloon, then use the space bar or enter key to select the balloon.  Use the arrow key to move the balloon to the right.  Keep going once the balloon its the wall.  The console will show that the live region is being updated to a blank value.

**Comment by terracoda** [comment]
@jbphet, with the query parameter, I do not see the console as you have described. However, using the visual PDOM, I see that the description, "At wall." is missing. I think this is related to an issue tracking that the incorrect grid model was being used (will add issue here).

The latest updates to the descriptions (especially regarding locations) are not yet implemented. It is about to undergo an overhaul.

Descriptions do indicate that the balloon is at the wall, "At wall", "At upper wall", or "At lower wall."

**Comment by jessegreenberg** [comment]
Thanks all, this is no longer an issue.  Accessible content has been reimplemented since this issue was created, there are no 'blank' alerts coming through the alert queue any more.
